src/winston2.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as is from '@sindresorhus/is';
+
+import * as common from './common';
+import * as types from './types/core';
+
+// cant have type checking on winston because it may be version 2 or 3
+const winston = require('winston');
```

Comment:
Would it still be possible to annotate this object with the minimal interface needed for this file? I assume that such an interface won't change radically since 3.x has already superseded 2.x.

Reply:
happy to hear about examples if you want to lend a moment to look over my shoulder.